_observing-your-data/ad/managing-anomalies.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: default
+title: Managing anomalies
+nav_order: 60
+parent: Anomaly detection
+has_children: false
+---
+
+## Alert on anomalies
+
+You can create an [Alerting monitor]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/) using either the Anomaly detector editor or the Extraction query editor. When you want to monitor an individual anomaly detector's results and notification condition thresholds on anomaly grade and confidence, use the Anomaly detector editor. Otherwise, use the Extraction query editor to monitor multiple detectors' results or write complex queries/trigger conditions.
+
+<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection/alerting_editor.png" alt="Alerting editor" width="800" height="800">
```

Comment:
This shows an "Edit monitor" page but on line 11 we're saying "create an alerting monitor". Should this image show the "Create monitor" page?

Reply:
yes, let me replace the image